### PR TITLE
Pandas/matplotlib test fixes

### PR DESCRIPTION
**Pull request by charlesreid1**
This PR resolves several warnings raised during tests that come from Pandas and matplotlib.

### CODE REVIEW

**Comment by charlesreid1** [comment]
This PR is a purely "cosmetic" change to the tests - it does not change any answers, it simply cleans up many of the warnings that are currently cluttering Travis output.

File: trackpy/motion.py [hunk 1]
```diff
@@ -268,14 +268,18 @@ def compute_drift(traj, smoothing=0, pos_columns=None):
...
+    # since it is required for computing the average
+    # delta_x/delta_y (and possibly delta_z) for each frame.
     f_diff.rename(columns={'frame': 'frame_diff'}, inplace=True)
     f_diff['frame'] = f_sort['frame']
 
     # Compute the per frame averages. Keep only deltas of the same particle,
     # and between frames that are consecutive.
     mask = (f_diff['particle'] == 0) & (f_diff['frame_diff'] == 1)
-    dx = f_diff.loc[mask, pos_columns + ['frame']].groupby('frame').mean()
+    f_diff_fix = f_diff.rename_axis('index')
```
Comment (caspervdw) [inline trackpy/motion.py]: Why is this necessary? Please add a comment in the code.

File: trackpy/tests/test_feature_saving.py [hunk 2]
```diff
@@ -16,6 +16,11 @@
 from trackpy.tests.common import StrictTestCase
 from pims import ImageSequence
 
+# Quiet warnings about get_store being deprecated.
+# These come from pandas.io and are caused by line 62:
+#       s = self.storage_class(STORE_NAME)
+import warnings
+warnings.filterwarnings("ignore",message="get_store is deprecated")
```
Comment (caspervdw) [inline trackpy/tests/test_feature_saving.py]: PEP style coding, please add a space after the comma separating arguments.

**Comment by caspervdw** [comment]
Thanks @charlesreid1 ! I left some minor notes.

File: trackpy/motion.py (hunk 1 above)
Comment (charlesreid1) [reply trackpy/motion.py]: Good catch! These lines were not included intentionally with this commit. Will roll them back in another commit.
